src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx

```diff
@@ -36,16 +37,17 @@ class IntroductionPage extends React.Component {
             </li>
             <li className="process-step list-three">
               <div><h5>VA Review</h5></div>
-              <p>We review feedback in the order we receive it. We may contact you if we need more information from you.</p>
-              <p>If we need to get in touch with you, we’ll contact you from an email that will look like this: process.vbavaco@va.gov.</p>
+              <p>We’ll review your feedback and then send it to the school. We may contact you if we need more information from you.</p>
+              <p>Feedback that isn’t related to VA education benefits may be sent to another agency for review. We’ll get back to you within 45 days to let you know how we’re handling your feedback.</p>
             </li>
             <li className="process-step list-four">
-              <div><h5>VA Follow Up</h5></div>
-              <p>We’ll get back to you within 45 days to let you know how we’re handling your feedback and if we’ve had any communication with your school. Feedback that isn’t related to VA education benefits may be sent to another agency for review.</p>
+              <div><h5>School Follow Up</h5></div>
+              <p>The school will review your feedback and send us their response.</p>
             </li>
             <li className="process-step list-five">
-              <div><h5>School Follow Up</h5></div>
-              <p>The school will review your feedback. We’ll send you the school’s response and ask you if you think their response resolves your issue. In some cases, the school may get in touch with you directly to discuss your feedback.</p>
+              <div><h5>VA Follow Up</h5></div>
```

Comment:
I need to check to see if we actually need to wrap these `h5` tags in `div` tags

Reply:
Good question, let's revisit this. I will set up a todo to check this in a subsequent PR.